tests/python/PycodestyleBearTest.py

```diff
@@ -22,6 +22,12 @@ def hello():
     valid_files=(good_file,),
     invalid_files=(bad_file,))
 
+PycodestyleBearTest = verify_local_bear(
```

Comment:
Nah keep it. Actually those tests do different things, one relies on the tool-defaults, one relies on the coafile defaults.